sdks/java/build-tools/src/main/resources/beam/suppressions.xml

```diff
@@ -99,7 +99,8 @@
   <suppress id="ForbidNonVendoredGrpcProtobuf" files=".*extensions.*sql.*datastore.*" />
   <suppress id="ForbidNonVendoredGrpcProtobuf" files=".*zetasql.*DateTimeUtils\.java" />
   <suppress id="ForbidNonVendoredGrpcProtobuf" files=".*zetasql.*ZetaSqlBeamTranslationUtils\.java" />
-  <suppress id="ForbidNonVendoredGrpcProtobuf" files=".*zetasql.*ZetaSQLDialectSpecTest\.java" />
```

Comment:
It looks like there are several other classes named `ZetaSQL...`. I filed BEAM-10457 for cleanup.

Reply:
Thanks!